package.json

```diff
@@ -22,6 +22,8 @@
     "@fortawesome/react-fontawesome": "0.1.17",
     "axios": "0.26.1",
     "babel-plugin-macros": "3.1.0",
+    "camelcase-keys": "7.0.2",
+    "date-fns": "2.28.0",
```

Comment:
Note to reviewer: Prepare date library for next PR, to reduce the next PR file changes